types/types.go

```diff
@@ -243,6 +243,7 @@ type Info struct {
 	ServerVersion      string
 	ClusterStore       string
 	ClusterAdvertise   string
+	SecurityOptions    string
```

Comment:
@thaJeztah Yes `[]string` is definitely better than a space separated string and easy to process. Let me update the pull request. Thanks!